### PR TITLE
Repository Configuration

### PR DESCRIPTION
**Pull request by waldirio**
Configured the module responsible for the repos *EPEL, Puppet 5 and Foreman*

### CODE REVIEW

**Review by phess — APPROVED** [review]

File: ansible/tasks/_prep-repos.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+#
```
Comment (tchellomello) [inline ansible/tasks/_prep-repos.yml]: Let's remove this empty line

File: ansible/tasks/_prep-repos.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+# 
+
+  - name: Install Puppet 5
+    yum: 
+      name: https://yum.puppetlabs.com/puppet5/puppet5-release-el-7.noarch.rpm
+      state: present
+    tags:
+      - install_puppet_5
```
Comment (tchellomello) [inline ansible/tasks/_prep-repos.yml]: how about if we create a generic tag? like (enable_repository)?

File: ansible/tasks/_prep-repos.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+
+  - name: Install EPEL 7
+    yum: 
+      name: http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+      state: present
+    tags:
+      - install_epel_7
```
Comment (tchellomello) [inline ansible/tasks/_prep-repos.yml]: same as above

File: ansible/tasks/_prep-repos.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+  - name: Install Puppet 5
+    yum: 
+      name: https://yum.puppetlabs.com/puppet5/puppet5-release-el-7.noarch.rpm
+      state: present
+    tags:
+      - install_puppet_5
+
+
+  - name: Install EPEL 7
```
Comment (tchellomello) [inline ansible/tasks/_prep-repos.yml]: Should we test if the box is compatible with RPM?

File: ansible/tasks/_prep-repos.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      state: present
+    tags:
+      - install_epel_7
+
+  - name: Install Foreman
+    yum: 
+      name: https://yum.theforeman.org/releases/1.16/el7/x86_64/foreman-release.rpm
+      state: present
+    tags:
+      - install_foreman
```
Comment (tchellomello) [inline ansible/tasks/_prep-repos.yml]: same as above

**Review by tchellomello — CHANGES_REQUESTED** [review]
Added some comments to be checked.

**Review by tchellomello — APPROVED** [review]
Looks good!!

**Pull request merged**